content/Default/apps/content.json

```diff
@@ -2031,6 +2031,44 @@
     "tryexec": ""
   },
   {
+    "application-id": "com.endlessm.spanish_guarani_dict",
+    "category": "Resources",
+    "core": false,
+    "custom-splash-screen": "com.endlessm.spanish_guarani_dict-splash.jpg",
+    "description": "Este diccionario bilingüe Castellano-Guaraní y Guaraní-Castellano es muy didáctico, pues está especialmente desarrollado para los estudiantes y docentes. Se rige por estrictas normas de la educación básica. Cuenta con indicación de la categoría gramatical (adj., s., v.,) y con 8.132 entradas.",
+    "displayShape": "b1",
+    "exec": "com.endlessm.spanish_guarani_dict",
+    "featured_img": "",
+    "icon": "com.endlessm.spanish_guarani_dict-icon.png",
+    "is_featured": false,
+    "is_offline": false,
+    "personalities": [
+      "All"
+    ],
+    "screenshots": {
+      "C": [
+        "com.endlessm.spanish_guarani_dict-screenshot1.jpg",
+        "com.endlessm.spanish_guarani_dict-screenshot2.jpg",
+        "com.endlessm.spanish_guarani_dict-screenshot3.jpg"
+      ],
+      "es": [
+        "com.endlessm.spanish_guarani_dict-screenshot1.jpg",
+        "com.endlessm.spanish_guarani_dict-screenshot2.jpg",
+        "com.endlessm.spanish_guarani_dict-screenshot3.jpg"
+      ],
+      "pt": [
+        "com.endlessm.spanish_guarani_dict-screenshot1.jpg",
+        "com.endlessm.spanish_guarani_dict-screenshot2.jpg",
+        "com.endlessm.spanish_guarani_dict-screenshot3.jpg"
+      ]
+    },
+    "splash-screen-type": "Custom",
+    "square_img": "com.endlessm.spanish_guarani_dict-thumb.jpg",
+    "subtitle": "Diccionario bilingüe de Castellano-Guaraní y Guaraní-Castellano",
+    "title": "Diccionario Bilingüe | Ñe’ẽ Ayvu",
```

Comment:
@rmacqueen @MarielaBalbona Do we have any better title for this? This title won't fit under the desktop icon.